test/test_ops.py

```diff
@@ -317,7 +318,8 @@ def fn(*inputs, **kwargs):
                             fusible_nodes = op.autodiff_fusible_nodes
 
                         self.assertAutodiffNode(traced_fn.last_graph, op.assert_autodiffed, nonfusible_nodes, fusible_nodes)
-                        self.assertAutodiffNode(script_fn.last_graph, op.assert_autodiffed, nonfusible_nodes, fusible_nodes)
+                        if op.test_scripted_jit:
+                            self.assertAutodiffNode(script_fn.last_graph, op.assert_autodiffed, nonfusible_nodes, fusible_nodes)
```

Comment:
why are we not applying this check for tracing?

Reply:
Tracing JIT works fine here. Scripted JIT doesn't work because it cannot pick up the custom `op.get_op`, as the script is built using `op.name`.